ocean_lib/models/erc721_factory.py

```diff
@@ -124,12 +126,16 @@ def start_multiple_token_order(self, orders: list, from_wallet: Wallet) -> str:
 
     def create_nft_with_erc(
         self,
-        nft_create_data: Union[dict, tuple, CreateERC721DataNoDeployer],
+        nft_create_data: Union[
+            dict, tuple, CreateERC721DataNoDeployer, CreateERC721Data
+        ],
         erc_create_data: Union[dict, tuple, CreateErc20Data],
         from_wallet: Wallet,
     ) -> str:
         # TODO: this will be handled in web3 py
-        if isinstance(nft_create_data, CreateERC721DataNoDeployer):
+        if isinstance(nft_create_data, CreateERC721DataNoDeployer) or isinstance(
```

Comment:
there is no argument for additional erc20 deployer in that ABI, we should not support regular CreateERC721Data, just CreateERC721DataNoDeployer. This applies everywhere where it was changed. I wouldn't have created a different named tuple if both would have the same ABI structure.